### DISCUSSION TITLE
Change wording in "Offering" screen of Offer Builder

### DISCUSSION THREAD

**Discussion by danieljperry** [Chia-Network/chia-blockchain-gui#1087] [Ideas]
Currently (1.6.1 rc2) in the Offer Builder screen, the `Offering` side says `Assets I own that I'd like to trade for`. The "for" at the end is incorrect. Request to change this to `Assets I would like to trade` and on the Request side say `Assets I would like to receive` (or something similar)

**Comment by BrandtH22** [comment]
this has been updated in a previous version of the GUI
